### PR TITLE
models for products

### PR DESCRIPTION
**Pull request by Marifer-Enriquez**

### CODE REVIEW

File: client/.env [hunk 1]
```diff
@@ -1 +1,4 @@
-NODE_PATH=./src
+DB_USERNAME=root
```
Comment (jibm82) [inline client/.env]: @Marifer-Enriquez I think we shouldn't be performing changes to this file. The one for setting up the env variables should be in the root folder.

File: models/Product.js [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
       type: DataTypes.STRING,
       allowNull: false,
       validate: {
         len: [1]
       }
     },
-    body: {
-      type: DataTypes.TEXT,
+    sku: {
+      type: DataTypes.NUMBER,
```
Comment (jibm82) [inline models/Product.js]: @Marifer-Enriquez I think this field should be a `STRING`.

File: models/Product.js [hunk 3]
```diff
@@ -1,23 +1,36 @@
...
       type: DataTypes.STRING,
-      defaultValue: "Personal"
-    }
+      allowNull: false
+      validate: {
+        len: [1]
+      }
+    },
+    epc: {
+      type: DataTypes.NUMBER,
```
Comment (jibm82) [inline models/Product.js]: @Marifer-Enriquez Same here

File: routes/apiProductRoutes.js [hunk 4]
```diff
@@ -1,84 +1,87 @@
...
 
-//   // GET route for getting all of the posts
-//   app.get("/api/posts/", function(req, res) {
-//     db.Post.findAll({})
-//       .then(function(dbPost) {
-//         res.json(dbPost);
-//       });
-//   });
+  // GET route for getting all of the posts
+  app.get("/api/posts/", function(req, res) {
```
Comment (jibm82) [inline routes/apiProductRoutes.js]: @Marifer-Enriquez these routes are all pointing to `/api/posts/...` I believe there should be referencing products. Maybe we should rename the paths.

File: routes/apiProductRoutes.js [hunk 5]
```diff
@@ -1,84 +1,87 @@
...
+    db.Post.findAll({})
```
Comment (jibm82) [inline routes/apiProductRoutes.js]: @Marifer-Enriquez There is no `Post` model, I think we should be referencing the `Product` model. I guess we should update that reference.

File: models/Product.js (hunk 2 above)
Comment (Marifer-Enriquez) [reply models/Product.js]: done!

File: routes/apiProductRoutes.js (hunk 4 above)
Comment (Marifer-Enriquez) [reply routes/apiProductRoutes.js]: done!

File: models/Product.js (hunk 3 above)
Comment (Marifer-Enriquez) [reply models/Product.js]: done!

File: routes/apiProductRoutes.js (hunk 5 above)
Comment (Marifer-Enriquez) [reply routes/apiProductRoutes.js]: done!

**Pull request merged**